pizzaserver-server/src/main/java/io/github/willqi/pizzaserver/server/network/protocol/packets/AnimateEntityPacket.java

```diff
@@ -0,0 +1,80 @@
+package io.github.willqi.pizzaserver.server.network.protocol.packets;
+
+import com.google.common.primitives.UnsignedInteger;
+import com.google.common.primitives.UnsignedLong;
+import com.google.common.primitives.UnsignedLongs;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class AnimateEntityPacket extends ImplBedrockPacket {
+
+    public static final int ID = 0x9e;
+
+    private String animation;
+    private String nextState = "default";
+    private String stopExpression = "query.any_animation_finished";
+    private String controller;
+    private float blendOutTime = 0;
+    private ArrayList<UnsignedLong> entityRuntimeIDs = new ArrayList<>();
+
```

Comment:
I feel like a Set would be better here since you can't have duplicate entity ids

Reply:
Also is there a specific reason for using the unsigned long rather than just a long directly? I don't believe you have to convert it on your side. The VarInts writing the longs should be good enough I think